consensus/src/main/java/org/apache/iotdb/consensus/IConsensus.java

```diff
@@ -44,15 +44,61 @@ public interface IConsensus {
   ConsensusReadResponse read(ConsensusGroupId groupId, IConsensusRequest IConsensusRequest);
 
   // multi consensus group API
-  ConsensusGenericResponse addConsensusGroup(ConsensusGroupId groupId, List<Peer> peers);
 
-  ConsensusGenericResponse removeConsensusGroup(ConsensusGroupId groupId);
+  /**
+   * Require the <em>local node</em> to create a Peer and become a member of the given consensus
+   * group. This node will prepare and initialize local statemachine {@link IStateMachine} and other
+   * data structures. After this method returns, we can call {@link #addPeer(ConsensusGroupId,
+   * Peer)} to notify original group that this new Peer is prepared to be added into the latest
+   * configuration.
+   *
+   * @param groupId the consensus group this Peer belongs
+   * @param peers other known peers in this group
+   */
+  ConsensusGenericResponse createPeer(ConsensusGroupId groupId, List<Peer> peers);
```

Comment:
createPeer should be called on a node that does not contain any peer of the consensus group, to avoid one node having more than one replica.

Reply:
done